util/build_configs/build_configs.py

```diff
@@ -92,6 +92,7 @@ def validate(self):
     ( 'unwind',             'CHPL_UNWIND', ),
     ( 'mem',                'CHPL_MEM', ),
     ( 'atomics',            'CHPL_ATOMICS', ),
+    ( 'gmp',                'CHPL_GMP', ),
```

Comment:
This is going to greatly lengthen your build time.  Do you really need it, or would some other setting work?  Frankly I'd be inclined to use "none" here, since these are just preliminary builds we're doing to see if the technique of having sites build their own modules even works.